src/test/scala/nl.knaw.dans.easy.deposit/docs/DatasetXmlSpec.scala

```diff
@@ -0,0 +1,202 @@
+          <ddm:available>2018-03-14</ddm:available>
+          <ddm:audience>D35200</ddm:audience>
+          <ddm:audience>D33000</ddm:audience>
+          <ddm:accessRights>OPEN_ACCESS</ddm:accessRights>
+        </ddm:profile>
+        <ddm:dcmiMetadata>
+          <dcterms:alternative xml:lang="nld">alternative title 1</dcterms:alternative>
+          <dcterms:alternative xml:lang="nld">alternative title2</dcterms:alternative>
+          <dcx-dai:creatorDetails>
+            <dcx-dai:author>
+              <dcx-dai:titles xml:lang="nld">Dr.</dcx-dai:titles>
+              <dcx-dai:initials>O.</dcx-dai:initials>
+              <dcx-dai:insertions>van</dcx-dai:insertions>
+              <dcx-dai:surname>Belix</dcx-dai:surname>
+            </dcx-dai:author>
+          </dcx-dai:creatorDetails>
+          <dcx-dai:creatorDetails>
+            <dcx-dai:author>
+              <dcx-dai:organization>
+                <dcx-dai:name xml:lang="nld">my organization</dcx-dai:name>
+              </dcx-dai:organization>
+            </dcx-dai:author>
+          </dcx-dai:creatorDetails>
+          <dcterms:rightsholder>
+            <dcx-dai:author>
+              <dcx-dai:role>RightsHolder</dcx-dai:role>
+              <dcx-dai:organization>
+                <dcx-dai:name xml:lang="nld">rightsHolder1</dcx-dai:name>
+              </dcx-dai:organization>
+            </dcx-dai:author>
+          </dcterms:rightsholder>
+          <dcterms:rightsholder>
+            <dcx-dai:author>
+              <dcx-dai:titles xml:lang="nld">Dr.</dcx-dai:titles>
+              <dcx-dai:initials>A.S.</dcx-dai:initials>
+              <dcx-dai:insertions>van</dcx-dai:insertions>
+              <dcx-dai:surname>Terix</dcx-dai:surname>
+              <dcx-dai:role>RightsHolder</dcx-dai:role>
+            </dcx-dai:author>
+          </dcterms:rightsholder>
+          <dcterms:publisher xml:lang="nld">pub1</dcterms:publisher>
+          <dcterms:publisher xml:lang="nld">pub2</dcterms:publisher>
+          <dc:source xml:lang="nld">source1</dc:source>
+          <dc:source xml:lang="nld">source2</dc:source>
+          <dcterms:dateCopyrighted>2018-03-18</dcterms:dateCopyrighted>
+          <dcterms:valid>2018-03-17</dcterms:valid>
+          <dcterms:modified>2018-02-02</dcterms:modified>
+          <dcterms:issued>Groundhog day</dcterms:issued>
+          <dcterms:dateSubmitted>2018-03-22</dcterms:dateSubmitted>
+          <dcterms:license>http://creativecommons.org/publicdomain/zero/1.0</dcterms:license>
+        </ddm:dcmiMetadata>
+      </ddm:DDM>
+
+    val datasetMetadata = DatasetMetadata(readTestData("datasetmetadata-from-ui-all.json"))
+      .getOrElse(fail("could not parse test data"))
+    val actual = DatasetXml(datasetMetadata).getOrElse(fail("conversion to DDM failed"))
+
+    prettyPrinter.format(actual) shouldBe prettyPrinter.format(expected)
+  }
+
+  "DDM validation" should "succeed for the minimal json" in {
+    assume(triedSchema.isSuccess)
```

Comment:
Note the variation on the usual [assume(canConnect)](https://github.com/DANS-KNAW/easy-solr4files-index/blob/f4f3c59cf212b83a037422ecac22d865beee3689/src/test/scala/nl.knaw.dans.easy.solr4files/TestSupportFixture.scala#L83-L95). 

In this case the **test** is performing the schema-validation, in the linked case the **tested code** does the validation. In this case we don't have the custom time out and the ignored test will take significantly longer when working offline. But adding the `canConnect` code would add an extra second when the schema's are accessible.

This might be the place to start experimenting with local copies of the full tree of schema's to get less dependent on external services, but that is beyond the scope of this issue.